source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,75 @@
+
+Unlike the rest of the taxonomy only content associated with [specific organisations][spec-orgs] is allowed to be tagged to world taxons (DFID, FCO, UKVI, DIT, DExEU) and only [particular document types][specific-doc-types] (guidance, forms, detailed guides, and document collections) may be tagged to them. Presumably this is because the world pages are intended to be curation mechanisms for FCO and similar rather than content tagged to a topic.
+
+They are rendered in a completely different format to regular topic taxons. Compare https://www.gov.uk/education/running-and-managing-a-school with https://www.gov.uk/world/yemen.
+
+[taxonomy]: https://docs.publishing.service.gov.uk/manual/taxonomy.html
+[spec-orgs]: https://github.com/alphagov/whitehall/blob/56006c6f6ba033fbe450ef91d46204499e62e337/config/worldwide_tagging_organisations.yml
+[specific-doc-types]: https://github.com/alphagov/whitehall/blob/master/app/models/edition/taggable_organisations.rb#L4-L9
+
+## Problems with world taxonomy
+
+There have been a number of protracted discussions about whether this concept represents [technical debt for GOV.UK][tech-debt] tech community or if it is a form of product debt. Their existence is not well known among people who work on GOV.UK, despite it being a knowledge prerequisite to do any development work with the regular topic taxonomy. Any apps pulling taxons out of the Publishing API will get both so need to filter down to the ones they need. This means that all apps that use the taxonomy have to make concessions for the world taxonomy even if they don't use it.
+
+The rules regarding which departments and document types world taxons apply to are enforced by Whitehall but not by content-tagger meaning that these rules are not enforced uniformly.
+
+We also have a format of [world location][world-location] that was the predecessor to the world taxon pages. This is still used for a number of things ([atom feed][atom-feed], tagging in Whitehall, rendering the [world page][world-page]) so we’ve effectively got two concepts of the same thing and need to always keep them in sync.
+
+Content associated with a world location is tagged to the previous concept of a world location rather than a world taxon. For example https://www.gov.uk/government/news/information-and-events-for-uk-nationals-living-in-portugal has no data link to https://www.gov.uk/world/portugal and instead only has a link to the now defunct Portugal world location page. A web link for the taxon is then [created][create-link] based on the URL.
+
+World taxons are treated the same as topic taxons in content tagger, the only clue a user has that they are dealing with something different is the /world part of their path.
+
+Categories haven't been nurtured or refined so the world taxon structure is dense and repetitive. This results in a poor user experience for anyone tagging to it via Whitehall for example.
```

Comment:
I think this is missing a "The" at the start. It might need some qualifying information too as I think we introduce a new term here of categories and we may way want to think about using an existing term we've used like structure. For example "The structure of the world taxonomy is dense and repetitive.

Reply:
👍